index.js

```diff
@@ -3983,6 +3983,26 @@
   S.splitOn =
   def('splitOn', {}, [$.String, $.String, $.Array($.String)], splitOn);
 
+  //# splitOnRegex :: RegExp -> String -> Array String
+  //.
+  //. Returns the substrings of its second argument separated by evaluated
+  //. matches of its first argument.
```

Comment:
@davidchambers I'd say then it should be made completely clear that this is just a "safe" wrapper, i.e.

> Type-checked version of `String#split` that accepts RegEx as a separator.
>
> Takes a pattern and a string, and returns the result of splitting the string at every non-overlapping occurrence of the pattern (removing the matched substrings, depending on a pattern).
>
> [Full documentation on MDN.](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/split)